server/models/user.model.js

```diff
@@ -287,31 +287,31 @@ module.exports = function(mongoose) {
 
         let result;
 
-        try {
-          result = await firebaseApp.auth().signInWithEmailAndPassword(email, password);
-        } catch (err) {
-          //TODO: If test user, check if error is because user doesn't exist and if so, create the user (in firebase)
-          Log.error(err);
-          return false;
-        }
+        // try {
+        //   result = await firebaseApp.auth().signInWithEmailAndPassword(email, password);
+        // } catch (err) {
+        //   //TODO: If test user, check if error is because user doesn't exist and if so, create the user (in firebase)
+        //   Log.error(err);
+        //   return false;
+        // }
 
         const query = {
-          email: result.user.email.toLowerCase(),
+          email: email.toLowerCase(),
```

Comment:
I think this is also part of the local changes to your system. If so then let's not merge this.